locale/te.yml

```diff
@@ -1468,12 +1503,14 @@ __TRAIT__: ""
     # 👀 @Ln11211
     # ✅ @KarthikMothiki "సంపూర్ణ సంఖ్య"
     # ✅ @sujithkumardola
+    # ✅ @sh1fu-X
   all: "అన్ని"
     # 👀 @nagendrady
     # 👀 @manojkrishnak
     # 👀 @Ln11211
     # ✅ @KarthikMothiki "అన్ని"
     # ✅ @sujithkumardola
+    # ✅ @sh1fu-X
   any: "ఏదైనా"
```

Comment:
would you mind adding your review for `any` as well? It's easy to accidentally skip a few 😄